src/ui/components/Transfer.js

```diff
@@ -57,6 +79,16 @@ export const Transfer = ({
   const [receiver, setReceiver] = useState<Account | null>(null);
   const [amount, setAmount] = useState<string>("");
   const [memo, setMemo] = useState<string>("");
+  const isXSmall = useMediaQuery((theme) => theme.breakpoints.down("xs"));
+
+  const disabled = (amount) => {
+    return (
+      !Number(amount) ||
+      !(sender && receiver) ||
+      gt(fromFloatString(amount), sender.balance) ||
+      lt(fromFloatString(amount), fromFloatString("0"))
+    );
+  };
```

Comment:
Why does this need to be a function if it's only used once? Can just do:

```suggestion
  const isDisabled = (
      !Number(amount) ||
      !(sender && receiver) ||
      gt(fromFloatString(amount), sender.balance) ||
      lt(fromFloatString(amount), fromFloatString("0"))
    );
```

Reply:
Unsure, is there a benefit between making it a function rather than a const?